authentication/auth.css

```diff
@@ -0,0 +1,105 @@
+.authentication-page{
+    height: 100vh; 
+    position: relative; 
+}
+
+.form-container{   
+    display: flex;
+    align-items: center;
+    justify-content: center;       
+}
+
+.form-container .h3{
+    color: var(--primary-color);
+    padding-bottom: 1rem;
+}
+
+.authentication-form-container{     
+    margin-top: 1.88rem;
+    z-index: 1;     
+}
+
+.input-with-icon{
+    position: relative;
+}
+
+.input-with-icon > input{
+    max-width: 23rem;
+}
+
+.input-icon > input{
+    max-width: 21.88rem;
+}
+
+.icons-right{
+    position: absolute;
+    right: 1.56rem;
+    bottom: 1.43rem;
```

Comment:
Minor suggestion: You can use shorthand property here for less lines of code, can refactor later